[MRG] Update pybv to write non-voltage channels (#670)
mne-tools/mne-bids: doc/whats_new.rst

Requirements
^^^^^^^^^^^^

> - Updated requirements for ``pybv`` to 0.5 allowing BrainVision written files to contain all types of non-voltage channels, by `Adam Li`_ (:gh:`670`)

Reviewer: ```suggestion
- For writing BrainVision files, ``pybv`` version 0.5 is now required to allow writing of non-voltage channels, by `Adam Li`_ (:gh:`670`)
```
